### ISSUE TITLE
Translation memory

### ISSUE THREAD

**Issue by comradekingu** [sledilnik/website#630]
I tried translating https://hosted.weblate.org/projects/sledilnik/website/nb_NO/ and there is a lot of repetition.

https://hosted.weblate.org/user/kingu/ is not an admin, but alternatively that can be set up in https://hosted.weblate.org/access/sledilnik/

https://hosted.weblate.org/memory/project/sledilnik/
Use shared translation memory [x]

Should make it visible in https://hosted.weblate.org/projects/settings/sledilnik/#workflow

https://hosted.weblate.org/settings/sledilnik/website/#translation
Allow spreading translation [x]

Can mark stuff as needing edit when propagated/spread/imported.

Between https://hosted.weblate.org/projects/iso-codes/ and https://hosted.weblate.org/projects/weblate/languages/
there should be enough language names/countries to import/use.

**Comment by stefanb** [comment]
Shared translation memory is enabled:
![image](https://user-images.githubusercontent.com/319826/96551047-09091180-12b2-11eb-9682-89589aee2e1b.png)

There are some older versions from the times when project renamed few times when being set-up initially in weblate:
![image](https://user-images.githubusercontent.com/319826/96550591-5a64d100-12b1-11eb-9a26-2da742c755b4.png)

Can/should we do anything else?

**Comment by stefanb** [comment]
When i translated i remember translations of languages and country names being suggested from other projects, presumably via shared translation memory.

**Comment by comradekingu** [comment]
Yes, but they can be added automatically, either as suggestions, marked as needing edits, or translated, or reviewed
https://hosted.weblate.org/settings/sledilnik/website/#translation I think, but I'll look for it in the plugins.

**Comment by stefanb** [comment]
I added automatic translation addon, but according to 
https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-autotranslate-autotranslate
> This addon is triggered automatically when new strings appear in a component.
